client/src/components/FlightSearch.js

```diff
@@ -73,68 +71,70 @@ const FlightSearch = ({ submit }) => {
   const handleTravellers = (event) => setTravellers(event.target.value);
 
   const handleSubmit = async (event) => {
-  	event.preventDefault();
-	  console.log("testing")
-  	const fromCity = fromCities.find((city) => from === city.PlaceName).PlaceId;
-  	const toCity = toCities.find((city) => to === city.PlaceName).PlaceId;
-  	const formattedDepartureDate = formatDate(departureDate);
-  	const formattedReturnDate = formatDate(returnDate);
-  	if (from && to && departureDate && returnDate) {
-  		if (returnDate >= departureDate) {
-  			const response = await fetch(
-  				`api/flights/quotes/${fromCity}/${toCity}/${formattedDepartureDate}/?inboundDate=${formattedReturnDate}`
-  			);
-  			const data = await response.json();
-  			if (data.status === 200) {
-  				submit({
-  					date: departureDate.toString(),
-  					flights: data,
-  				});
+    event.preventDefault();
+    const fromCity = fromCities.find((city) => from === city.PlaceName).PlaceId;
+    const toCity = toCities.find((city) => to === city.PlaceName).PlaceId;
+    const formattedDepartureDate = formatDate(departureDate);
+    const formattedReturnDate = formatDate(returnDate);
+    const response = await fetch(
+      `api/flights/quotes/${fromCity}/${toCity}/${formattedDepartureDate}/?inboundDate=${formattedReturnDate}`
+    );
+    const data = await response.json();
+	console.log(data)
 
-				  console.log(response)
-  			} else if ("from" in data) {
-  				setFromError(true);
-  			} else if ("to" in data) {
-  				setToError(true);
-  			} else if ("outboundDate" in data) {
-  				setDepartDateError(true);
-  			} else {
-  				setReturnDateError(true);
-  			}
-  		} else {
-  			setReturnDateError(true);
-  		}
-  		// return date is optional
-  	} else if (from && to && departureDate) {
-  		const response = await fetch(
-  			`api/flights/quotes/${from}/${to}/${formattedDepartureDate}`
-  		);
-  		const data = await response.json();
-  		if (data.status === 200) {
-  			submit({
-  				date: departureDate.toString(),
-  				flights: data,
-  			});
+    // if (from && to && departureDate && returnDate) {
+    // 	if (returnDate >= departureDate) {
+    // 		const response = await fetch(
+    // 			`api/flights/quotes/${fromCity}/${toCity}/${formattedDepartureDate}/?inboundDate=${formattedReturnDate}`
+    // 		);
+    // 		const data = await response.json();
+    // 		if (data.status === 200) {
+    // 			submit({
```

Comment:
Delete the code you're commenting out. If you need to get it back, you can always do that with `git` if it was part of a previous commit.

Reply:
Will effect those changes now. Forgot to pull from dev before pushing.